packages/calcite-components/src/components/action/action.tsx

```diff
@@ -115,6 +135,14 @@ export class Action extends LitElement implements InteractiveComponent {
   /** Indicates whether the text is displayed. */
   @property({ reflect: true }) textEnabled = false;
 
+  /**
+   * Specifies the type of the action.
+   * - `"button"`: A standard button action.
+   * - `"toggle"`: An action that can switch between pressed and unpressed states.
+   * - `"expand"`: An action specifically for indicating if a control is expanded or collapsed.
```

Comment:
- When would I as a user use "expand" vs "toggle" - is tit that "expand" would be for Actions that correspond to say, open Panels in a Shell Panel - and "toggle" would be for an Action that invoke tooltips or Popovers, etc?

Reply:
Toggle would be for things that just need to convey an active/pressed state and don't have any need for "aria-controls" (menu, expanded, etc)

For example, a toolbar "bold" button would be a "toggle" type whereas an action menu button would be an "expand" type.